encoding.opam

```diff
@@ -38,5 +38,6 @@ build: [
 dev-repo: "git+https://github.com/formalsec/encoding.git"
 available: (arch = "x86_64" | arch = "arm64") & os != "win32" & arch != "x86_32"
 pin-depends: [
-  [ "colibri2.dev" "git+https://git.frama-c.com/pub/colibrics.git#5.1.0"]
+  [ "colibri2.dev" "git+ssh://git@git.frama-c.com/pub/colibrics.git#5.1.0"]
```

Comment:
I don't think it will work it the `ssh` link, the CI says it cannot fetch it: 

```
could not synchronize /home/runner/work/encoding/encoding/_opam/.opam-switch/sources/colibri2 from "git+ssh://git@git.frama-c.com/pub/colibrics.git#5.1.0":
```

Reply:
I can see that, but I don't think reviewing every commit as they arrive is very helpful, you can take over the PR if you want, or wait for me to fix it to review, if I don't manage to do it I'll let you know :)

Reply:
Oh so sorry, ofc I'll stop :)